### DISCUSSION TITLE
Content Sync- Can we fetch history of the content sync activity.

### DISCUSSION THREAD

**Discussion by raj13799** [Adobe-Consulting-Services/acs-aem-commons#3396] [General]
Is there a logger to check the content sync activity to tell us the information about the frequent activities(which user, paths and completion status). Are there any documentation to review it? Thanks!!

**Comment by YegorKozlov** [comment]
The current implementation does not log which user/path were updated. I had something like this in mind when implementing content sync, but didn't get around to implementing it. 

I'd say the tool could create an audit node under `/var/acs-commons/contentsync` and put the entire console output into it, i.e. what you see in the UI will be saved in the audit node along with basic properties like userId, etc...

**Reply by raj13799** [reply]
@YegorKozlov - Thanks for the update. Verified the node under /var/acs-commons/contentsync, so far i don't see it is capturing the details in the audit node. Is it something we have to customize?

**Comment by YegorKozlov** [comment]
@raj13799 I was sure I replied to this and today found my answer stuck in draft :)

The feature I'm talking about does not yet exist. It's something we can do in futuire releases.

**Reply by YegorKozlov** [reply]
The Sync history feature was added in 6.7.0
